aries-site/src/themes/aries.js

```diff
@@ -0,0 +1,110 @@
+import { hpe } from 'grommet-theme-hpe';
+import { deepMerge } from 'grommet/utils';
+
+export const aries = deepMerge(hpe, {
+  name: 'HPE 2',
+  rounding: 6,
+  spacing: 24,
+  defaultMode: 'dark',
+  global: {
+    colors: {
+      brand: 'green',
+      background: {
+        dark: '#1A1F2B',
+        light: '#FFFFFF',
+      },
+      'background-strong': {
+        dark: '#000000',
+        light: '#FFFFFF',
+      },
+      'background-weak': {
+        dark: '#44444480',
+        light: '#E8E8E880',
+      },
+      'background-xweak': {
+        dark: '#66666699',
+        light: '#CCCCCC90',
```

Comment:
I thought HEX color should be represented with 6 characters, is this coming from the designer? 
(the selected colors has 8 characters)

Reply:
Interesting! well, they might need to change the name from HEX to OCT now...

Reply:
Yes. @taysea is correct.

@ShimiSun Are you going to lead the lobbying effort for the name change? ;)

Reply:
lol, someone should :)